openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/TestSuiteScheduler.tsx

```diff
@@ -38,12 +38,14 @@ const TestSuiteScheduler: React.FC<TestSuiteSchedulerProps> = ({
   return (
     <Row gutter={[16, 32]}>
       <Col span={24}>
-        <CronEditor
-          includePeriodOptions={includePeriodOptions}
-          isQuartzCron={isQuartzCron}
-          value={repeatFrequency}
-          onChange={(value: string) => setRepeatFrequency(value)}
-        />
+        <Form data-testid="schedule-container">
+          <CronEditor
+            includePeriodOptions={includePeriodOptions}
+            isQuartzCron={isQuartzCron}
```

Comment:
Don't we need to remove this flag now?

Reply:
this will be false initially, but will remove it.